docs/concepts/vega-chain/network.md

```diff
@@ -130,10 +125,18 @@ See a full list of the network parameters used for PoW in the [spam protection p
 | spam.pow.increaseDifficulty           | If a wallet exceeds the max transactions per block, the difficulty of the PoW increases by this factor | <NetworkParameter frontMatter={frontMatter} param="spam.pow.increaseDifficulty" hideName={true} />
 | spam.pow.numberOfPastBlocks         	| To compute the wallet transaction quota, transactions can be assigned to past blocks; this parameter defines how far back that goes | <NetworkParameter frontMatter={frontMatter} param="spam.pow.numberOfPastBlocks" hideName={true} />
 
+
+## Loading network state: Snapshots
+To allow a Vega node to be restarted without the need to replay the whole blockchain, a Vega node can load an existing snapshot created by a different node, which will populate all the network state. The node can then resume listening to blocks after the snapshot, until it gets to the live block height, at which point it will be able to contribute to the chain.
+
+:::note Try it out
+Node operators can refer to [how to use snapshots](../../node-operators/how-to/use-snapshots.md) for a step-by-step guide on loading network state with snapshots. 
+:::
+
 ## Network restarts: Checkpoints
```

Comment:
"Checkpoints" I like to say LNL Checkpoints to make it even more different from Snapshots to disambiguate.